### PR TITLE
Subgraph tests use SDK-Core

### PR DESCRIPTION
**Pull request by 0xdavinchee**

### CODE REVIEW

File: packages/subgraph/package.json [hunk 1]
```diff
@@ -40,13 +40,14 @@
...
         "generate-sf-meta-local": "COMMIT_HASH=local CONFIGURATION=local BRANCH=local TAG=local ./tasks/getSFMeta.sh",
+        "generate-ethers-types": "typechain --target=ethers-v5 --out-dir=typechain './abis/*.json'",
         "check-updates": "ncu --target minor --dep prod,dev"
     },
     "dependencies": {
         "@decentral.ee/web3-helpers": "^0.5.3",
         "@graphprotocol/graph-cli": "0.25.1",
         "@graphprotocol/graph-ts": "0.24.1",
-        "@superfluid-finance/js-sdk": "0.5.10",
+        "@superfluid-finance/sdk-core": "^0.2.0",
```
Comment (hellwolf) [inline packages/subgraph/package.json:50]: per monorepo ethic, you should just use 0.2.0 instead of '^0.2.0'

Comment (hellwolf) [reply packages/subgraph/package.json:50]: how about this

**Review by hellwolf — CHANGES_REQUESTED** [review]
Great, surprised it got migrated without many changes. Left couple of package related change requests. Otherwise good to go.

image

File: packages/subgraph/package.json (hunk 1 above)
Comment (0xdavinchee) [reply packages/subgraph/package.json]: using 0.2.0 and removed truffle and ganache-time-traveler

**Pull request merged**